core/src/it/scala/io/km8/core/kafka/KafkaProducerSpec.scala

```diff
@@ -16,19 +15,19 @@ import org.apache.kafka.clients.consumer.OffsetResetStrategy
 
 object KafkaProducerSpec extends DefaultRunnableSpec:
 
-  private val consumerLayer: ZLayer[Has[KafkaContainer], Nothing, Has[KafkaConsumer]] =
-    Clock.live ++ Blocking.live ++ itlayers.clusterConfig(clusterId = "cluster_id") >>> KafkaConsumer.liveLayer
+  private val consumerLayer: ZLayer[KafkaContainer, Nothing, KafkaConsumer] =
```

Comment:
The same comment here: should we try automatic layer creation without operators?

Reply:
not sure why would that be necessary, if possible I would use operators to speed up compile time